kms/api/QuickStart/QuickStart.cs

```diff
@@ -31,40 +28,22 @@ public class QuickStart
         public static void Main(string[] args)
         {
             // Your Google Cloud Platform project ID.
-            string projectId = "YOUR-PROJECT-ID";
+            string projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");

```

Comment:
Quick starts don't read environment variables.  We expect the user to replace the string in the code.

Reply:
Sorry, got tripped up on this because `dotnet test` was failing.  I see now that this gets worked around in a Powershell script.  It would be nice if `dotnet test` would work out of the box, but understood that the convention is what it is for now.